src/audio/drc/drc_math_lib/asin/c_fixpt/drc_asin_fixpt.c

```diff
@@ -0,0 +1,234 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+
+/* Include Files */
+#include "drc_asin_fixpt.h"
+
+/* Type Definitions */
+#include "stdint.h"
+
+/* Function Declarations */
+static int32_T asr_s32(int32_T u, uint32_T n);
```

Comment:
these are still using the wrong types (lower case t)

Reply:
OK - I change int32_T to lower case int32_t